src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/ScannedDocumentsHelper.java

```diff
@@ -44,6 +47,61 @@ private ScannedDocumentsHelper() {
             .collect(toList());
     }
 
+    public static void setExceptionRecordIdToScannedDocuments(
+        ExceptionRecord exceptionRecord,
+        CaseUpdateDetails caseDetails
+    ) {
+        @SuppressWarnings("unchecked")
+        var caseData = (Map<String, Object>) caseDetails.caseData;
+        List<ScannedDocument> scannedDocuments = getScannedDocuments(caseData);
+
+        if (exceptionRecord.scannedDocuments == null) {
```

Comment:
we would aways have scanned documents and even if we wanted to validate this would be done when we would receive the callback request.

Reply:
wanted to be bulletproof but ok

Reply:
Agreed should be validated but should be at entry level so that we fail fast